doc/lua_api.txt

```diff
@@ -4555,7 +4555,8 @@ These functions return the leftover itemstack.
     * `param2` overrides `facedir` and wallmounted `param2`
     * `prevent_after_place`: if set to `true`, `after_place_node` is not called
       for the newly placed node to prevent a callback and placement loop
-    * returns `itemstack, success`
+    * returns `itemstack, position`
```

Comment:
`minetest.item_place` relies on these return values, which means its docs need to be changed too

and this code too:
https://github.com/minetest/minetest/blob/8e757859d6a6bf6482480904e8485e9344e567ab/builtin/game/item.lua#L417-L418

Reply:
Done, I think